### ISSUE TITLE
Generated cookbook metadata needs updating

### ISSUE THREAD

**Issue by tas50** [chef/cookstyle#169]
### What category of cop is this?:

Please try to select as few as possible.

- [ ] Deprecation - A cop that alerts to a deprecation in the Chef Infra Client
- [ ] Correctness - A cop that alerts when incorrect coding behavior is being used
- [ ] Sharing - A cop that alerts to missing functionality necessary for sharing cookboooks with others
- [ ] Style - A cop that alerts to a style best practice
- [ ] Simplification - A cop that alerts to ways code can be simplified through refactoring
- [ ] Effortless - A cop that alerts on code that must be resolved to move to the Effortless pattern
- [ ] Policyfiles - A cop that alerts on code that must be resolved to move to the Policyfiles pattern

### Describe the new cop:
This warning is shown if you used knife cookbook create to create a new cookbook and didn’t override the maintainer and maintainer email. You need to set these to real values in metadata.rb or run knife again with the real values.

### What it would trigger on:

```ruby

```

**Comment by tyler-ball** [comment]
Should this be moved to be an issue against the `chef-cli` repo, or is it referring to updating the cops to verify the metadata?

**Comment by tas50** [comment]
@tyler-ball this is detecting when a user doesn't update those default fields and ships a cookbook. There are many on the Supermarket that still use the default values for author name and e-mail.
